8.x/core-concepts/cruds.md

```diff
@@ -2,99 +2,55 @@
 
 [[toc]]
 
-This document reviews how to implement CRUD (create, read, update, delete) in Larammerce.
-As you know, it is not only related to Larammerce and is related to the structure of the Laravel framework.
-In larammerce, tools have been created to make implement in CRUD easier. Also, the created CRUD should be the same as the previous CRUDs of the system, so it should be followed the guidelines provided in this document.
+This document reviews how to implement `CRUD` (create, read, update, delete) in the Larammerce platform. A `ToDoList` project is done step by step to illustrate the operational concepts.
```

Comment:
I would like to have some explanations about cruds here. Readers should have basic information about the nature of the concept you are going to propose. As a sample:

In programming, CRUD is an acronym that stands for Create, Read, Update, and Delete. These are the four basic functions that a computer program typically performs when working with data in a database or file system.